src/run.py

```diff
@@ -63,14 +65,16 @@ def main():
     # Get the number of chapters
     print('How many chapters should the book have?')
     chapters = int(input('> '))
+    if chapters <= 1:
+        words = 1
 
     # Get the number of words per chapter
     print('How many words should each chapter have?')
     # Check if it is below 1200
```

Comment:
this comment probably needs to be updated as well, but I also found it weird to have min limit...

Reply:
I fooled around with a lot of the settings just testing what didn't work and I needed this to prevent something unwanted from happening. Right now the code works without all prompting because I have each setting fixed right now. I made it up to 10 chapters with 2000 words using the newer November gpt-3.5-turbo